acceptance/framework/environment/environment.go

```diff
@@ -59,6 +61,20 @@ func NewKubernetesEnvironmentFromConfig(config *config.TestConfig) *KubernetesEn
 		kenv.contexts[SecondaryContextName] = NewContext(config.SecondaryKubeNamespace, config.SecondaryKubeconfig, config.SecondaryKubeContext)
 	}
 
+	// Optionally, deploy apps into a separate namespace.
```

Comment:
This is true and I wasn't sure about putting this here, but your comment doesn't feel like sufficient justification because we are already associating namespaces with the existing contexts?

I would need to rework this anyway though to match the recent changes to support N clusters.

Reply:
I did remove the app namespace flags, and no longer use a separate context - so resolving this thread (but leaving others open for discussion)